static/js/containers/LearnerSearchPage_test.js

```diff
@@ -104,7 +100,38 @@ describe('LearnerSearchPage', function () {
         TestUtils.Simulate.click(saveButton);
       }).then(() => {
         assert.isFalse(helper.store.getState().ui.emailDialogVisibility);
-        assert.isTrue(sendSearchResultMail.calledWith("subject", "body", {abc: 123}));
+        assert.isTrue(sendSearchResultMail.calledWith("subject", "body", searchkit.buildQuery().query));
+      });
+    });
+  });
+
+  it('filters names if there is text in the textbox', () => {
```

Comment:
a more accurate description might be `'sends a request to find users by name when text is entered into the search box'`